jetty-util/src/main/java/org/eclipse/jetty/util/resource/ResourceCollection.java

```diff
@@ -169,25 +212,38 @@ public void setResourcesAsCSV(String csvResources)
                     " argument must be a string containing one or more comma-separated resource strings.");
         }
         
-        List<Resource> resources = new ArrayList<>();
+        List<Resource> res = new ArrayList<>();
         
         try
         {            
             while(tokenizer.hasMoreTokens())
             {
-                Resource resource = Resource.newResource(tokenizer.nextToken().trim());
-                if(!resource.exists() || !resource.isDirectory())
-                    LOG.warn(" !exist "+resource);
-                else
-                    resources.add(resource);
+                String token = tokenizer.nextToken().trim();
+                // TODO: we should not trim here, as whitespace is valid for paths
```

Comment:
Is a resource that is nothing but whitespace actually valid????

Reply:
Yup.

Shouldn't trim for paths like 'foo ' or ' bar' too.